spec/scripts/benchmarks.sh

```diff
@@ -9,21 +9,17 @@ set -exo pipefail
 
 IMAGE_NAME=${1:?"missing RUBY IMAGE NAME"}
 VERSION=$(echo "${IMAGE_NAME}" | cut -d":" -f2)
-REFERENCE_REPO=${2}
+OUTPUT_NAME=${2:?"missing THE OUTPUT NAME"}
+REFERENCE_REPO=${3}
 
 if [ -z "${REFERENCE_REPO}" ] ; then
   REFERENCE_REPO_FLAG=""
 else
   REFERENCE_REPO_FLAG="-v ${REFERENCE_REPO}:${REFERENCE_REPO}"
 fi
 
-## Transform the versions like:
```

Comment:
So the consumer will be the one providing the name so we don't need to have the same implementation in both places.